api/api-config-repos-v1/src/main/java/com/thoughtworks/go/apiv1/configrepo/ConfigReposControllerV1.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2018 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.thoughtworks.go.apiv1.configrepo;
```

Comment:
There's already a config repo api (in rails). Perhaps move this into a different package (and project) to avoid a conflict when the rails API is moved to java.